backend/src/main/java/com/huellapositiva/application/controller/ProposalApiController.java

```diff
@@ -328,6 +333,8 @@ public void submitProposalRevision(@PathVariable String id,
                     .path(PATH_ID).buildAndExpand(id)
                     .toUri();
             submitProposalRevisionAction.execute(id, dto, uri, accountId);
+        } catch (InvalidStatusIdException ex) {
```

Comment:
Did you meant InvalidProposalStatusException?

Reply:
It's done.